test/karma.conf.js

```diff
@@ -16,7 +16,7 @@ module.exports = function setKarmaConfig(config) {
     browsers: ['chromeHeadless'],
     browserDisconnectTimeout: 120000, // default 2000
     browserDisconnectTolerance: 1, // default 0
-    browserNoActivityTimeout: 300000, // default 10000
+    browserNoActivityTimeout: 6 * 60 * 1000, // default 10000
```

Comment:
Interesting that the original value seemed to have been motivated by Firefox already: https://docs.travis-ci.com/user/gui-and-headless-browsers/#karma-and-firefox-inactivity-timeouts.

At some point I'd like to debug why firefox is that much slower because you already see 2-3 slowdowns when profiling React (https://github.com/mui-org/material-ui/pull/24289). The goal being to confirm that this is only dev and not prod. 2-3 slower in prod is rough.

Reply:
Do you think that 6 minutes is enough?

Reply:
I have found the test config of karma: https://github.com/karma-runner/karma/blob/a14a24ef81348b29daf70e0ea8406588fdd01f19/test/client/karma.conf.js#L123-L129

Reply:
> Do you think that 6 minutes is enough?

I don't know. Intuitively, I think there's already something wrong when there's 5minutes of no activity. I would expect a passed/failed test to count as "activity".